### ISSUE TITLE
Overall design

### ISSUE THREAD

**Issue by alan-daniel** [dasilva333/TowerGhostForDestiny#66]
Have you considered hiring (or asking) a friendly graphic/web design guru for a more polished design? Don't get me wrong, I am totally grateful for being a part of the testing. It's just that the information looks kind of like a mess. The sections aren't very descriptive, it's hard to tell which character you're looking at, and it all feels like a super-long list of items instead of a true organizer. I know the subject material necessitates a lot of complex, redundant information, and it would be a work of genius to make a Destiny vault/gear manager that's "pretty"

I really hope you take this as constructive criticism, and not any kind of attack. Regardless, I hope to remain a user (and tester) for quite a while.

I also understand if your coding process is functionality-first, polish later -- I totally respect that; mine tends to be. If it is, disregard this post.

**Comment by dasilva333** [comment]
Yeah I agree the UI could use the help of a graphic designer, I posted a request on reddit when I started the project and nobody came through. Thankfully there's kind people like @joshhunt and @ssoriche  who simply contribute to the project without being asked and make small changes that make a big difference. I'm always willing to take in pull requests and I'm glad you posted this thread so I can makr it as help wanted and perhaps it'll inspire a designer to lend a hand.

Thanks

**Comment by bluetidepro** [comment]
I made a first step with this here: https://github.com/dasilva333/TowerGhostForDestiny/pull/150

I have lots of good ideas on how to improve the UI/UX! :)

**Comment by dasilva333** [comment]
Good stuff man thanks for the new icons, unfortunately it's not just the images under the www/assets folder that need change I also need the new icons replaced under www/res. Finally I need the source icon in 1024x1024 resolution to submit manually to the Apple App Store, if you fix up the pull request to include the www/res files I'll have no problem pulling it. Also make sure to include a separate link to the 1024px image in the comments not in the commit. Thanks for your help bro
